src/Microsoft.Tye.Hosting/HttpProxyService.cs

```diff
@@ -113,6 +113,8 @@ public async Task StartAsync(Application application)
 
                             builder.Configure(app =>
                             {
+                                app.UseWebSockets();
```

Comment:
This seems too easy; wouldn't it have been more fun to replace this entire proxy service with YARP? :-)

Reply:
This is my plan 😄